src/parser.rs

```diff
@@ -66,6 +67,12 @@ pub struct Function {
     pub body: Items,
 }
 
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+pub struct Table {
+    pub name: Ident,
+    pub body: Pipeline,
```

Comment:
(possibly `pipeline` would be more consistent, but your call)

Reply:
Thanks! OK!